images/scripts/update-cilium-envoy-image.sh

```diff
@@ -14,34 +14,42 @@ root_dir="$(git rev-parse --show-toplevel)"
 cd "${root_dir}"
 
 github_repo="cilium/proxy"
```

Comment:
wouldn't it be possible to even support forked github repos?

Reply:
Yes, added this change to support this:
```
--- a/images/scripts/update-cilium-envoy-image.sh
+++ b/images/scripts/update-cilium-envoy-image.sh
@@ -13,7 +13,7 @@ root_dir="$(git rev-parse --show-toplevel)"
 
 cd "${root_dir}"
 
-github_repo="cilium/proxy"
+github_repo=${proxy_repo:-"cilium/proxy"}
 github_branch=${proxy_branch:-"main"}
 
 latest_commit_sha="$(curl -s https://api.github.com/repos/${github_repo}/commits/"${github_branch}" | jq -r --exit-status '.sha')"
```

Reply:
nice, thanks